docs/reference/settings/security-settings.asciidoc

```diff
@@ -278,10 +278,13 @@ Defaults to `true`.
 The `type` setting must be set to `ldap`. In addition to the 
 <<ref-realm-settings>>, you can specify the following settings: 
 
-`url`:: Specifies one or more LDAP URLs in the format  
-`ldap[s]://<server>:<port>`. Multiple URLs can be defined using a comma 
-separated value or array syntax: `[ "ldaps://server1:636", "ldaps://server2:636" ]`. 
-`ldaps` and `ldap` URL protocols cannot be mixed in the same realm. Required.
+`url`::
+One or more LDAP URLs in the `ldap[s]://<server>:<port>` format. Required.
++
+To provide a single URL, use the `ldap` protocol: `ldap://server1:636`. To
+provide multiple URLs, use the `ldaps` protocol with an array syntax:
+`["ldaps://server1:636", "ldaps://server2:636" ]`. You can't mix the `ldap` and
+`ldaps` URL protocols.
```

Comment:
This is incorrect, though it's a logical interpretation of the old text (hence the need to update it)

Ignoring, for a moment, what we want the docs to say, the rules are:
- you can specific `ldap` or `ldaps` protocols for a single URL.
- you can specific `ldap` or `ldaps` protocols when providing multiple URLs, but you must be consistent across all URLs. That is, they must all be `ldap` or must all be `ldaps`.
- if providing multiple URLs you can either:
   1. use YAML array syntax `url: ["ldaps://server1:636", "ldaps://server2:636" ]`; _or_
   2. use a comma separated string: `url: "ldaps://server1:636,ldaps://server2:636"`

I struggle to find the sweet spot between documenting every possible option, and overwhelming the reader with too much information, vs just documenting a common set of option and have users get confused about why a certain behaviour is not documented.

Which is to say, I'm not sure whether it would be better to document all of the rules that I've written above, or simplify to it.